{{cookiecutter.app_name}}/README.md

```diff
@@ -12,17 +12,28 @@
 
 ### Development
 
+#### Create an ENV file
+
+To start development server, `.env` must be created.
+
+- Copy `.env.example` and rename to `.env`
```

Comment:
Should we change to this to make it clearer?
```suggestion
To start the development server, `.env` file must be created.

- Copy `.env.example` file and rename to `.env`
```

Reply:
Fixed in dd039c70dc12af74141df1f6dc5ba7acb1418307